afk-daily.sh

```diff
@@ -1484,8 +1486,7 @@ oakInnSpeedy() {
     until [ "$_oakInn_TRIES" -ge "$_oakInn_TRIES_MAX" ]; do
         until [ "$_oakInn_X_START" -ge "$_oakInn_X_END" ]; do
             # Tap on X coord to possibly collect present
-            input tap "$_oakInn_X_START" 1350
-            sleep 2
+            inputTapSleep "$_oakInn_X_START" 1350
```

Comment:
Since it was not the function `inputTapSleep`, it was not setting `screenshotRequired=true`, so if the first tap was not working, there was no more fresh screenshot to look at ;)

Reply:
Right... Can't believe I've been trying to fix this for various weeks now only to realise it's this tiny error... Thank you for the find!